device-types/Cisco/WS-C4507R+E.yaml

```diff
@@ -29,6 +22,6 @@ module-bays:
   - name: Slot 7
     position: '7'
   - name: PS1
-    position: '1'
```

Comment:
The issue with changing the module bay position, is that the module variable in the module template will only be expecting a number rather than PS1.